activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/IntegrationErrorReceivedEventHandler.java

```diff
@@ -58,6 +62,28 @@ public void handle(CloudRuntimeEvent<?, ?> event) {
         });
     }
 
+    @NotNull
+    private static List<StackTraceElement> addFullErrorMessageAsFirstStackTraceElement(
+        CloudIntegrationErrorReceivedEvent integrationEvent) {
+        StackTraceElement newTopStackTraceElement = getNewTopStackTraceElement(integrationEvent);
+        List<StackTraceElement> stackTraceElements = new java.util.ArrayList<>(List.of(newTopStackTraceElement));
```

Comment:
Why not to import the package for ArrayList?

Reply:
fixed